.travis.yml

```diff
@@ -46,7 +46,6 @@ matrix:
     - env: REQUIREMENTS=devel REXTRAS=all ES_VERSION=2.3.3
 
 python:
-  - "3.3"
```

Comment:
I removed 3.3 because [weakref](https://docs.python.org/3.3/library/weakref.html) was failing trying to call `weakref.finalize`, which I saw doesn't exist in that version.

Reply:
Ok, Invenio is not tested with python 3.3 anymore so we can say that it is not supported.